html-media-capture/all.html

```diff
@@ -3,39 +3,91 @@
+<tr class='subtest'><td>/html-media-capture/capture_image_missing-manual.html</td><td class='PASS'>PASS</td><td class='PASS'>PASS</td></tr>
+<tr class='test' id='test-file-9'><td><a href='http://www.w3c-test.org/html-media-capture/capture_image_user-manual.html' target='_blank'>/html-media-capture/capture_image_user-manual.html</a></td><td class='OK'>OK</td><td class='OK'>OK</td></tr>
+<tr class='subtest'><td>/html-media-capture/capture_image_user-manual.html</td><td class='PASS'>PASS</td><td class='PASS'>PASS</td></tr>
+<tr class='test' id='test-file-10'><td><a href='http://www.w3c-test.org/html-media-capture/capture_video_cancel-manual.html' target='_blank'>/html-media-capture/capture_video_cancel-manual.html</a></td><td class='OK'>OK</td><td class='OK'>OK</td></tr>
+<tr class='subtest'><td>/html-media-capture/capture_video_cancel-manual.html</td><td class='PASS'>PASS</td><td class='PASS'>PASS</td></tr>
+<tr class='test' id='test-file-11'><td><a href='http://www.w3c-test.org/html-media-capture/capture_video_environment-manual.html' target='_blank'>/html-media-capture/capture_video_environment-manual.html</a></td><td class='OK'>OK</td><td class='OK'>OK</td></tr>
+<tr class='subtest'><td>/html-media-capture/capture_video_environment-manual.html</td><td class='PASS'>PASS</td><td class='PASS'>PASS</td></tr>
+<tr class='test' id='test-file-12'><td><a href='http://www.w3c-test.org/html-media-capture/capture_video_invalid-manual.html' target='_blank'>/html-media-capture/capture_video_invalid-manual.html</a></td><td class='OK'>OK</td><td class='OK'>OK</td></tr>
+<tr class='subtest'><td>/html-media-capture/capture_video_invalid-manual.html</td><td class='PASS'>PASS</td><td class='PASS'>PASS</td></tr>
+<tr class='test' id='test-file-13'><td><a href='http://www.w3c-test.org/html-media-capture/capture_video_missing-manual.html' target='_blank'>/html-media-capture/capture_video_missing-manual.html</a></td><td class='OK'>OK</td><td class='OK'>OK</td></tr>
+<tr class='subtest'><td>/html-media-capture/capture_video_missing-manual.html</td><td class='PASS'>PASS</td><td class='PASS'>PASS</td></tr>
+<tr class='test' id='test-file-14'><td><a href='http://www.w3c-test.org/html-media-capture/capture_video_user-manual.html' target='_blank'>/html-media-capture/capture_video_user-manual.html</a></td><td class='OK'>OK</td><td class='OK'>OK</td></tr>
+<tr class='subtest'><td>/html-media-capture/capture_video_user-manual.html</td><td class='PASS'>PASS</td><td class='PASS'>PASS</td></tr>
+
       </table>
+
+<p><Strong>Failure analysis</strong>:
+<ol>
+  <li>
+    <a href='#test-file-2'>/capture_audio-manual.html</a><br>
+    <a href='#test-file-3'>/capture_audio_cancel-manual.html</a>:<br>
+    Capture is a generic mechanism across media types including image, video, audio, and possible other futher media types.<br>
```

Comment:
Spotted a minor typo after merge:

s/futher/further/

@Honry can you patch this?

Reply:
Whoops, I meant s/futher/future/ :-)